Sources/WebDriver/Element.swift

```diff
@@ -98,6 +98,16 @@ public struct Element {
         try session.findElements(startingAt: self, locator: locator, waitTimeout: waitTimeout)
     }
 
+    /// Finds an element using a given locator, starting from this element, and throwing upon failure.
+    /// - Parameter locator: The locator strategy to use.
+    /// - Parameter description: A human-readable description of the element, included in thrown errors.
+    /// - Parameter waitTimeout: The amount of time to wait for element existence. Overrides the implicit wait timeout.
+    /// - Returns: The element that was found.
+    @discardableResult // for use as an assertion
+    public func requireElement(locator: ElementLocator, description: String? = nil, waitTimeout: TimeInterval? = nil) throws -> Element {
```

Comment:
I was going to ask, actually, because I was thinking about how to put it in the "supported apis" documentation.  Could go either way.  We could also have a new target in this project that is "bonus stuff".

Reply:
I don't like (nonmonetary) bonuses. I think I'll move this to the arc repo. It'll also make more sense with the custom error type.